src/rpc/client.cpp

```diff
@@ -131,6 +131,8 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "setgenerate", 1 },
     { "setmintzerocoinstatus", 2 },
     { "setmintzerocoinstatus", 1 },
+    { "setsigmamintstatus", 2 },
```

Comment:
I think we don't need this since `setsigmamintstatus` have only 2 parameters and the only one that need to convert is the last one.